PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -2674,8 +2718,8 @@ function Get-SummaryReport
 
     $ClusterNodes = Import-Clixml (Join-Path $Path "GetClusterNode.XML")
     $Cluster = Import-Clixml (Join-Path $Path "GetCluster.XML")
-
-    $ClusterName = $Cluster.Name + "." + $Cluster.Domain
+	
+    $ClusterName = $Cluster.Name + "." + $ClusterDomain
```

Comment:
Where is ClusterDomain getting set during the SummaryReport? Try running Show-SddcDiagnosticReport on the produced ZIP package, I suspect this is relying on dynamic scoping when run during Get's invocation to create the summary report.